rewrite-weblogic/documentation/content/Migrate to Jakarta EE 9_1.md

```diff
@@ -0,0 +1,94 @@
+# Migrate to Jakarta EE 9.1
+**com.oracle.weblogic.rewrite.JakartaEE9_1**
+
+This recipe applies the changes required for migrating to Jakarta EE 9.1.
+
+### Tags:
+  - weblogic
+  - jakarta
+  - jakartaee
+
+### Recipe Source
+
+[jakarta-ee-9.1.yaml](https://github.com/oracle/rewrite-recipes/blob/main/rewrite-weblogic/src/main/resources/META-INF/rewrite/jakarta-ee-9.1.yaml)
+
+### Recipe List:
+
+  - [org.openrewrite.java.migrate.jakarta.JavaxMigrationToJakarta](https://docs.openrewrite.org/recipes/java/migrate/jakarta/javaxmigrationtojakarta)
+  - com.oracle.weblogic.rewrite.jakarta.Faces2xMigrationToJakarta3x
+  - com.oracle.weblogic.rewrite.jakarta.JavaxBeansXmlToJakarta9BeansXml
+  - com.oracle.weblogic.rewrite.jakarta.JavaxApplicationXmlToJakarta9ApplicationXml
+  - com.oracle.weblogic.rewrite.jakarta.JavaxEjbJarXmlToJakarta9EjbJarXml
+  - com.oracle.weblogic.rewrite.jakarta.RemovalsServletJakarta9
+  - com.oracle.weblogic.rewrite.jakarta.JavaxToJakartaCdiExtensions
+  - com.oracle.weblogic.rewrite.jakarta.UpdateJakartaPlatform9_1
+
+### Usage
+
+This recipe will migrate `javax` packages to `jakarta` and update Jakarta EE platform dependencies to 9.1.0.
+
+#### Maven POM
+
+1. Add the following to your Maven POM file.
+```
+<plugin>
+    <groupId>org.openrewrite.maven</groupId>
+    <artifactId>rewrite-maven-plugin</artifactId>
+    <version>5.43.0</version>
+    <configuration>
+        <exportDatatables>true</exportDatatables>
+        <activeRecipes>
+            <recipe>com.oracle.weblogic.rewrite.JakartaEE9_1</recipe>
+        </activeRecipes>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.weblogic.rewrite</groupId>
+            <artifactId>rewrite-weblogic</artifactId>
+            <version>0.4.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openrewrite.recipe</groupId>
+            <artifactId>rewrite-migrate-java</artifactId>
+            <version>2.28.0</version>
+        </dependency>
+    </dependencies>
+</plugin>
+...
+```
+2. Run `mvn rewrite:run` to run the recipe.
+
+#### Gradle
+
+1. Add the following to your `build.gradle` file:
+
+```
+plugins {
+    id("org.openrewrite.rewrite") version("7.1.4")
+}
+
+rewrite {
+    activeRecipe("com.oracle.weblogic.rewrite.JakartaEE9_1")
+    setExportDatatables(true)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    rewrite("org.openrewrite.recipe:rewrite-migrate-java:3.2.0")

```

Comment:
Is missing to add a line for `rewrite("com.oracle.weblogic.rewrite:rewrite-weblogic:0.4.2")`